extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css

```diff
@@ -611,7 +611,10 @@ md-progress-circular.md-default-theme .md-inner .md-left .md-half-circle
 }
 
 .parameter>div {
```

Comment:
yes, but turning the rule off has no effect. At least it seems so in my browser (chrome 58).